main/webapp/modules/core/scripts/dialogs/clustering-dialog.js

```diff
@@ -291,7 +291,7 @@ ClusteringDialog.prototype._cluster = function() {
 
     this._elmts.resultSummary.empty();
 
-    $.post(
+    var data = await $.post(
```

Comment:
Async-Await is used in different situations, considering we are still on native js. Async promises make no use over here else the changes are lgtm except the async-await block you added.